linux_os/guide/system/auditing/service_auditd_enabled/rule.yml

```diff
@@ -52,7 +52,7 @@ references:
     srg: SRG-OS-000037-GPOS-00015,SRG-OS-000038-GPOS-00016,SRG-OS-000039-GPOS-00017,SRG-OS-000040-GPOS-00018,SRG-OS-000041-GPOS-00019,SRG-OS-000042-GPOS-00021,SRG-OS-000051-GPOS-00024,SRG-OS-000054-GPOS-00025,SRG-OS-000122-GPOS-00063,SRG-OS-000254-GPOS-00095,SRG-OS-000255-GPOS-00096,SRG-OS-000365-GPOS-00152,SRG-OS-000392-GPOS-00172,SRG-OS-000480-GPOS-00227,SRG-OS-000062-GPOS-00031
     stigid@ol7: OL07-00-030000
     stigid@rhel7: RHEL-07-030000
-    stigid@rhel8: RHEL-08-010560
```

Comment:
We should probably tell DISA to remove one of these. We advised them to split the service auditd but there was already a STIG item that did that. The fix actually was just to remove the auditd service enabled from the install audit package STIG item.

Reply:
I'm preparing the documentation to send them with requests.